docs/configuring.rst

```diff
@@ -0,0 +1,60 @@
+================
+Configuration
+================
+
+There are tools to configure some of kopf functionality, like asynchronous
+tasks behaviour and logging events.
+
+.. note::
+    All configuration should be done before importing kopf modules.
+
+
+Configure logging events
+=================
```

Comment:
These "==="-lines should be at least the same length as the text they use as a caption/title: http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections

Reply:
fixed, thx